apps/admin/frontend/src/components/adjudication_ballot_image_viewer.tsx

```diff
@@ -84,63 +84,71 @@ const ZoomedOutBallotImageContainer = styled.div`
 const BallotImageViewerControls = styled.div<{ isZoomedIn: boolean }>`
   display: flex;
   justify-content: flex-end;
+  align-items: center;
   position: absolute;
   top: 0;
   z-index: 2;
   background: ${(props) => (!props.isZoomedIn ? 'rgba(0, 0, 0, 0.5)' : 'none')};
+  height: 4rem;
   width: 100%;
   padding: 0.5rem;
   gap: 0.5rem;
 `;
 
+function remToPx(rem: number) {
+  const fontSizeResult = safeParseNumber(
+    // trim 'px' from the result
+    getComputedStyle(document.documentElement).fontSize.substring(0, -2)
+  );
+  // fallback to default of 30
+  /* istanbul ignore next - fallback to default of 30 - @preserve */
+  const fontSize = fontSizeResult.isOk() ? fontSizeResult.ok() : 30;
```

Comment:
There's a lot of duplication here.

```suggestion
  /* istanbul ignore next - @preserve */
  const fontSize = fontSizeResult.ok() ?? 30;
```

Reply:
I see, I had thought we needed to check `isOk` first

And thanks for the push to write code that is self-descriptive, not needing additional comments. That's a principle that slipped my mind a bit as I was away from programming.